src/page/AddressStep.page.ts

```diff
@@ -0,0 +1,11 @@
+import { $, ElementFinder, promise } from 'protractor';
+
+export class AddressStepPage {
+ private get proccedToCheckout(): ElementFinder {
```

Comment:
add Button suffix

Reply:
Do i need to change the name proccedToCheckout --> addButtonSuffix() ??

Reply:
Don`t worry, i could understand, Just, I knew the word suffix.